### ISSUE TITLE
GitHub Action stuck on "starting your workflow run"

### ISSUE THREAD

**Issue by atc0005** [atc0005/elbow#107]
https://github.com/atc0005/elbow/commit/e244ccce30359de8b7129e39d05ba63fcabef392/checks?check_suite_id=286266038

I had thought this was entirely on GitHub's end (and it still might be), but digging around in the community forums I see several users note that typos in the configuration can lead to the same behavior:

- https://github.community/t5/GitHub-Actions/Stuck-on-quot-starting-your-workflow-run-quot/m-p/31702#M873
- https://github.com/actions/setup-ruby/issues/26

Others didn't seem to have any such typo:

- https://github.community/t5/GitHub-Actions/Action-blocked-on-quot-Starting-a-workflow-run-quot/m-p/33855#M1685
- https://github.com/goreleaser/godownloader/pull/142/files#annotation_6656311066563110

**Comment by atc0005** [comment]
See also #103, #106, #104, #105

**Comment by atc0005** [comment]
I suspect I found it:

```yaml
    runs-on: ${{ matrix.platform }}
    strategy:
      matrix:
        # Supported versions of Go
        go-version: [1.12.x, 1.13.x]

        # Supported LTS versions of Ubuntu Linux
        os: [ubuntu-16.04, ubuntu-18.04]
```

This line:

```yaml
    runs-on: ${{ matrix.platform }}
```

**Comment by atc0005** [comment]
Fixed by work on #108.
